### PR TITLE
Update Use cases and Manual testing examples

### PR DESCRIPTION
**Pull request by lirc572**
Preview of use cases: <https://lirc572.github.io/tp/DeveloperGuide.html#use-cases>

### CODE REVIEW

**Review by samuelfangjw — DISMISSED** [review]
not related to this  PR, but notice this sentence at the start of UC section

`(For all use cases below, the System is CoLAB and the Actor is the user, unless specified otherwise)`

Would it be better if the brackets were removed?

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -1018,44 +696,7 @@ document.addEventListener('DOMContentLoaded', () => {
...
-**Extensions**
-
-* 1a. The given project index is invalid.
-
-    * 1a1. CoLAB shows an error message.
-
-      Use case resumes at step 1.
-
-* 2a. The list of groupmates is empty.
+* 2a. The list of todos is empty.
```
Comment (vevek) [inline docs/DeveloperGuide.md]: list of todos/deadline

**Review by vevek — CHANGES_REQUESTED** [review]
Nice! only that minor change

**Review by vevek — APPROVED** [review]
LGTM!

**Pull request merged**